### PR TITLE
fix: Remove private true flag from  package json templates

### PR DESCRIPTION
**Pull request by kjose90**
#2155

Removed `private: true flag` from the package.json templates for EDMX and CAP applications from:

- `ui5-application-writer` package json template.
- `ui5-library-writer `template package json template.
- `ui5-library-writer` template typescript package json template.

Updated snapshots  under `fiori-elements-writer`, `fiori-freestyle-writer`, `ui5-application-writer` and `ui5-library-writer`

### CODE REVIEW

**Review by devinea — APPROVED** [review]
Removal of the private flag makes sense as we should not assume what people want to do with these modules. 
Snapshot updates.
patchset ✅

File: .changeset/famous-tigers-cough.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+'@sap-ux/fiori-freestyle-writer': minor
+'@sap-ux/ui5-application-writer': minor
+'@sap-ux/fiori-elements-writer': minor
+'@sap-ux/ui5-library-writer': minor
```
Comment (devinea) [inline .changeset/famous-tigers-cough.md:5]:
```suggestion
'@sap-ux/ui5-application-writer': minor
'@sap-ux/ui5-library-writer': minor
```
I think we can reduce the list here as FE and FF writers are just snapshot updates.

Comment (kjose90) [reply .changeset/famous-tigers-cough.md]: > I think we can reduce the list here as FE and FF writers are just snapshot updates.

Yes correct - Thanks

**Review by devinea — APPROVED** [review]
Thanks for the updates. Re-approving.

**Pull request merged**